README.md

```diff
@@ -91,15 +89,15 @@ git checkout -b new_branch_name
 2. Start developing!
 
 ```sh
-yarn start
+yarn dev
 ```
 
-- Open this directory in your favorite text editor / IDE, and see your changes live by visiting `localhost:8000` from your browser
+- Open this directory in your favorite text editor / IDE, and see your changes live by visiting `localhost:3000` from your browser
 - Pro Tip:
   - Explore scripts within `package.json` for more build options
   - Get **faster** local builds by building only one language. E.g. in your `.env` file, set `BUILD_LOCALES=en` to build the content only in English
```

Comment:
Not a blocker for this PR, but this tip is less helpful now that we use Next.js... it's handy if you're _building_ the site, but doesn't matter for `yarn dev` since pages are built on-demand.

Reply:
Agree. I found it informative though. Perhaps we could clarify that it is faster for production builds (`yarn build`).

Reply:
Changed the wording here to be production